packages/breadboard-web/src/boards/simplest.ts

```diff
@@ -4,42 +4,44 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Schema, base, board } from "@google-labs/breadboard";
-import { core } from "@google-labs/core-kit";
+import {
+  annotate,
+  board,
+  input,
+  object,
+  output,
+} from "@breadboard-ai/build";
+import { invoke } from "@google-labs/core-kit";
 
-const inputSchema = {
-  type: "object",
-  properties: {
-    text: {
-      type: "string",
-      title: "Prompt",
-      description: "The prompt to generate a completion for",
-      examples: ["Tell me a fun story about playing with breadboards"],
-    },
-  },
-  required: ["text"],
-} satisfies Schema;
+const text = input({
+  type: "string",
+  title: "Prompt",
+  description: "The prompt to generate a completion for",
+  examples: ["Tell me a fun story about playing with breadboards"],
+});
+
+const generator = input({
+  $id: "math-question",
+  title: "Generator",
+  type: annotate(object({}), {
+    behavior: ["board"],
+  }),
+  description: "The URL of the generator to call",
+  default: { kind: "board", path: "gemini-generator.json" },
+});
 
-const outputSchema = {
-  type: "object",
-  properties: {
-    text: {
-      type: "string",
-      title: "Response",
-      description: "The completion generated by the LLM",
-    },
-  },
-  required: ["text"],
-} satisfies Schema;
+const invoker = invoke({ $id: "gemini", $board: generator, instructions: text }).unsafeOutput("text");
 
-export default await board(() => {
-  return base
-    .input({ $id: "text", schema: inputSchema })
-    .to(core.invoke({ $id: "gemini", $board: "gemini-generator.json" }))
-    .to(base.output({ $id: "response", schema: outputSchema }));
-}).serialize({
+const results = output(invoker, {
+  title: "Response",
+  description: "The completion generated by the LLM",
+});
+
+export default board({
   title: "The simplest LLM-based board",
   description:
     "This board is as simple as it gets: takes text as input and invokes Gemini to generate a text response as output.",
   version: "0.0.3",
-});
+  inputs: { generator, text },
+  outputs: { results }
```

Comment:
It looks like the original board had an output called `text` with title `"Response"`. The new version instead has one called `results` with title `"Response"`.

I think we should either (1) use the same input/output names as the old version, or (2) use "response" because that seems more clear and matches the title.

Here's my suggestion:

```ts
export default board({
  title: "The simplest LLM-based board",
  description:
    "This board is as simple as it gets: takes text as input and invokes Gemini to generate a text response as output.",
  version: "0.0.3",
  inputs: { generator, text },
  outputs: {
    response: output(response, {
      title: "Response",
      description: "The completion generated by the LLM",
    }),
  }
});
```